src/App.vue

```diff
@@ -474,29 +468,13 @@ export default {
       if (business) {
         this.setEntityName(business.legalName)
         this.setEntityType(business.legalType)
-        this.setNextARDate(business.nextAnnualReport)
```

Comment:
The architecture here was wonky. This data is business entity info, which was set in the store, and which the BCOMP AR page would use.

I deleted this bit and now use `nextAnnualReport` from the task item (in todo list).